packages/mask/src/plugins/Tips/SNSAdaptor/TipsEntranceDialog.tsx

```diff
@@ -0,0 +1,359 @@
+import { cloneDeep } from 'lodash-unified'
+import { FC, useCallback, useEffect, useState } from 'react'
+import { useAsyncFn, useAsyncRetry } from 'react-use'
+import ActionButton from '../../../extension/options-page/DashboardComponents/ActionButton'
+import Services from '../../../extension/service'
+import { useI18N } from '../../../utils'
+import { getKvPayload, kvSet, useKvGet } from '../hooks/useKv'
+import { useProvedWallets } from '../hooks/useProvedWallets'
+import AddWalletView from './bodyViews/AddWallet'
+import SettingView from './bodyViews/Setting'
+import WalletsView from './bodyViews/Wallets'
+import Empty from './components/empty'
+import { VerifyAlertLine } from './components/VerifyAlertLine'
+import { WalletsByNetwork } from './components/WalletsByNetwork'
+export interface TipsEntranceDialogProps {
+    open: boolean
+    onClose: () => void
+}
+const useStyles = makeStyles()((theme) => ({
+    walletBtn: {
+        fontSize: '14px',
+    },
+    alertBox: {
+        marginBottom: '20px',
+    },
+    actions: {
+        position: 'absolute',
+        bottom: 16,
+        width: 'calc( 100% - 32px)',
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+        gap: theme.spacing(1.5),
+    },
+    dContent: {
+        height: 600,
+        position: 'relative',
+        boxSizing: 'border-box',
+    },
+    btnContainer: {
+        width: '100%',
+        display: 'flex',
+        flexDirection: 'row-reverse',
+    },
+    loading: {
+        position: 'absolute',
+        top: '50%',
+        left: '50%',
+        transform: 'translate(-50%,-50%)',
+    },
+}))
+
+enum BodyViewStep {
+    Main = 'Tips',
+    Setting = 'Settings',
+    Wallets = 'Wallets',
+    AddWallet = 'Add Wallet',
+}
+
+interface WalletButtonProps extends ButtonProps {
+    step: BodyViewStep
+}
+
+const WalletButton: FC<WalletButtonProps> = ({ step, onClick }) => {
+    const { classes } = useStyles()
+    if (step === BodyViewStep.AddWallet) return null
+    return (
+        <div className={classes.btnContainer}>
+            <Button onClick={onClick} className={classes.walletBtn} variant="contained" size="small">
+                {step === BodyViewStep.Wallets ? BodyViewStep.AddWallet : BodyViewStep.Wallets}
+            </Button>
+        </div>
+    )
+}
+
+export function TipsEntranceDialog({ open, onClose }: TipsEntranceDialogProps) {
+    const { t } = useI18N()
+    const { classes } = useStyles()
+    const [showAlert, setShowAlert] = useState(true)
+    const [bodyViewStep, setBodyViewStep] = useState<BodyViewStep>(BodyViewStep.Main)
+    const [hasChanged, setHasChanged] = useState(false)
+    const [rawPatchData, setRawPatchData] = useState<BindingProof[]>([])
+    const [rawWalletList, setRawWalletList] = useState<BindingProof[]>([])
+    const supportedNetworks = useActivatedPlugin(PluginId.Tip, 'any')?.enableRequirement?.web3?.[
+        NetworkPluginID.PLUGIN_EVM
+    ]?.tipsSupportedChains
```

Comment:
Don't add this. You can use `Object.keys(enableRequirement.web3) as NetworkPluginID`.